examples/cfq/train.py

```diff
@@ -80,56 +80,59 @@ def create_model(vocab_size: int) -> nn.Module:
 
 
 def cross_entropy_loss(logits: jnp.array, labels: jnp.array,
-                       lengths: jnp.array):
+                       lengths: jnp.array, vocab_size: int):
   """Returns cross-entropy loss."""
+  labels = common_utils.onehot(labels, vocab_size)
```

Comment:
Ah I see, so you moved the OHE into this loss function, so you can remove it in the other places. Nice!